apps/glusterfs/node_entry_test.go

```diff
@@ -22,12 +22,11 @@ import (
 )
 
 func createSampleNodeEntry() *NodeEntry {
-	g := utils.GenUUID()
 	req := &api.NodeAddRequest{
 		ClusterId: "123",
 		Hostnames: api.HostAddresses{
-			Manage:  []string{"manage" + g[:4] + g[24:]},
-			Storage: []string{"storage" + g[:4] + g[24:]},
+			Manage:  []string{"manage" + utils.GenUUID()},
+			Storage: []string{"storage" + utils.GenUUID()},
```

Comment:
you could have just used `g` without truncation here. But I guess it's ok to waste 2 uuids on this. ;-)